docs/future-work/learning-module-improvements/deal-with-moving-objects.md

```diff
@@ -1,3 +1,9 @@
 ---
 title: Deal with Moving Objects
 ---
+
+This work relates to first being able to [Detect Local and Global Flow](../../future-work/sensor-module-improvements/detect-local-and-global-flow.md).
+
+Our current idea is to then use this information to model the state of the object, such that beyond its current pose, we also capture how it is moving as a function of time. This information can then be made available to other learning modules for voting and hierarchical processing.
```

Comment:
Yeah I guess it just feels like velocity could be something more fundamental, i.e. that might be part of all CMP messages once we have flow etc. working? I.e. that things like color are optional, but whether something is moving or not is as central as its pose? But might not end up being so.

Reply:
Hm yeah interesting question. I'm not sure how velocity would apply to more abstract objects/concepts but maybe it can be. I was just thinking if it is one of many optional features we get maximum flexibility but maybe that's not needed.